cypress/e2e/profile.cy.ts

```diff
@@ -150,3 +150,25 @@ describe('Checks the analytics switch', () => {
     });
   });
 });
+
+describe('Checks the language switch', () => {
+  beforeEach(() => {
+    cy.setUpApi({
+      currentMember: {
+        ...currentMember,
+      },
```

Comment:
Why use spreading here ? 
```suggestion
      currentMember,
```